core/math/aabb.cpp

```diff
@@ -1,5 +1,5 @@
 /*************************************************************************/
-/*  rect3.cpp                                                            */
+/*  aabb.cpp                                                            */
```

Comment:
There's one additional space needed to align the `*/` at the end of the line :P

Reply:
Ah so clang-format can't detect those